vitrina/datasets/templates/vitrina/datasets/coretable.html

```diff
@@ -149,6 +149,20 @@
             <td>{{ dataset.get_access_rights_display }}</td>
         </tr>
     {% endif %}
+    {% if dataset.applicable_legislation.exists %}
+        <tr>
+            <th>{% translate "Teisinis pagrindas" %}</th>
+            <td>
+                <ul>
+                    {% for legislation in dataset.applicable_legislation.all %}
+                        {% with label=legislation.description|default:legislation.url %}
+                            <li><a href="{{ legislation.url }}" target="_blank" title="{{ label }}">{{ label|truncatechars:30 }}</a></li>
+                        {% endwith %}
```

Comment:
```suggestion
                        <li><a href="{{ legislation.url }}" target="_blank" title="{{ label }}">{{ legislation | truncatechars:30 }}</a></li>
```

`ApplicableLegislation.__str__()` method already handles description or url